reference/physics.md

```diff
@@ -112,6 +112,9 @@ or stacking simulation, you will need to specify the [Physics](#physics) nodes
 for the boxes so that the friction and gravity forces are applied to these
 objects.
 
+Note that if you add a dynamic [Solid](#solid) node inside a passive object, then it will behave like a normal dynamic object independent from the parent one.
```

Comment:
This sentence is not very clear to me...

It is not clear what you mean by "passive". Do you mean "kinematic" (e.g., a differential wheel robot without physics) or "static" (the static environment)?